components/audio_hal/driver/es8388/es8388.c

```diff
@@ -346,6 +346,10 @@ es8388_init (audio_hal_codec_config_t *cfg)
                        0x80); // set internal ADC and DAC use the same LRCK
                               // clock, ADC LRCK as internal LRCK
   res |= es_write_reg (ES8388_ADDR, ES8388_DACCONTROL23, 0x00); // vroi=0
+  res |= es_write_reg (ES8388_ADDR, ES8388_DACCONTROL24, 0x1E); // Set L1 R1 L2 R2 volume. 0x00: -30dB, 0x1E: 0dB, 0x21: 3dB
```

Comment:
How does this impact other boards using the same codec (lyrat v4.3, etc.)?

Reply:
They probably should be fine with this but I'm not sure since I don't have one.

Reply:
Ok I'll test this and compare